zilencer/models.py

```diff
@@ -84,6 +84,35 @@ def __str__(self) -> str:
         return f"{self.server!r} {self.user_id}"
 
 
+class RemoteRealm(models.Model):
+    """
+    Each object corresponds to a single remote Realm that is using the
+    Mobile Push Notifications Service via `manage.py register_server`.
+    """
+
+    server = models.ForeignKey(RemoteZulipServer, on_delete=models.CASCADE)
+
+    # The unique UUID and secret for this realm.
+    uuid = models.UUIDField(unique=True)
+    uuid_owner_secret = models.TextField()
+
+    subdomain = models.TextField()
+
+    # The fields below are analogical to RemoteZulipServer fields.
+
+    last_updated = models.DateTimeField("last updated", auto_now=True)
+
+    # Whether the realm registration has been deactivated.
+    deactivated = models.BooleanField(default=False)
```

Comment:
I guess if we're sending `deactivated` from the remote server, we would need a different name for that; maybe `realm_deactivated` would be appropriate as a place to store that, distinct from the registration being deactivated?

Reply:
Named these `realm_deactivated` and `registration_deactivated`